internal/gapicgen/cmd/genbot/generate.go

```diff
@@ -157,12 +157,9 @@ func hasChanges(dir string) (bool, error) {
 	inmem := &bytes.Buffer{}
 	w := io.MultiWriter(os.Stderr, inmem)
 
-	c := exec.Command("bash", "-c", "git status --short")
+	c := command.Create("bash", "-c", "git status --short")
```

Comment:
What do you think of naming the package `vexec` or `execv`? I like `execv` more because it doesn't sound like "very exec. much shell. :dog: "

I like that more than naming it `command` because then we can rename `Create` to `Command` and it becomes a drop-in replacement for `os/exec`, just like `execabs`.

Reply:
good idea, execv it is!